### ISSUE TITLE
AWS: take default AMI ID from datacenter configuration, if available

### ISSUE THREAD

**Issue by kdomanski** [kubermatic/dashboard#1176]
**What would you like to be added? / User Story**:
Similarly to how it is being already done for VSphere and OpenStack since #689, we'd like to be able to fill out default images from the datacenter config for AWS.

blocked by: kubermatic/kubermatic#3168
